dream-big-api/app/api/category_api.rb

```diff
@@ -11,12 +11,14 @@ class CategoryApi < Grape::API
 
   desc 'Allow creation of a category'
   params do
+    requires :id, type: Integer, desc: 'Category ID'
     requires :name, type: String, desc: 'Category name'
     requires :description, type: String, desc: 'The description of the category'
   end
   post '/category' do
     category_parameters = ActionController::Parameters.new(params)
       .permit(
+        :id,

```

Comment:
You don't need an ID to create a category, as the ID is created by the database when the category is created.

Reply:
Thank you for the feedback. ID parameter has been removed from category_api.rb